multimodal-understanding/getting-started/06_tool_calling.ipynb

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
<p>nit: it's better to run through lint. the comment is too long.</p>

<pre class="ql-syntax" spellcheck="false">                        "type": "object", # The top level schema MUST have a type of "object", properities and required keys. No other fields are allowed at this level
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-nova-samples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>